app/views/insights/_contributors.html.erb

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
I think this part of code should be in application.html.erb.

Reply:
solved